server/gc/src/main/java/org/apache/accumulo/gc/GarbageCollectWriteAheadLogs.java

```diff
@@ -244,36 +251,53 @@ private long removeTabletServerMarkers(Map<UUID,TServerInstance> uidMap,
     return result;
   }
 
+  private long removeFile(Path path) {
+    try {
+      if (!useTrash || !fs.moveToTrash(path)) {
+        fs.deleteRecursively(path);
+      }
+      return 1;
+    } catch (FileNotFoundException ex) {
+      // ignored
```

Comment:
If this can't happen, it should probably throw an AssertionError. Can this normally happen?

Could also consider adding a debug or trace log here, whether or not it's normal, because it could indicate problems (instead of throwing an AssertionError).

Reply:
I think logging a debug message would make sense